app/scripts/controllers/transactions/tx-state-manager.js

```diff
@@ -10,27 +11,21 @@ import {
 import { getFinalStates, normalizeTxParams } from './lib/util'
 
 /**
-  TransactionStateManager is responsible for the state of a transaction and
-  storing the transaction
-  it also has some convenience methods for finding subsets of transactions
-  *
-  *STATUS METHODS
-  <br>statuses:
-  <br>   - `'unapproved'` the user has not responded
-  <br>   - `'rejected'` the user has responded no!
-  <br>   - `'approved'` the user has approved the tx
-  <br>   - `'signed'` the tx is signed
-  <br>   - `'submitted'` the tx is sent to a server
-  <br>   - `'confirmed'` the tx has been included in a block.
-  <br>   - `'failed'` the tx failed for some reason, included on tx data.
-  <br>   - `'dropped'` the tx nonce was already used
-  @param {Object} opts
-  @param {Object} [opts.initState={ transactions: [] }] initial transactions list with the key transaction {array}
-  @param {number} [opts.txHistoryLimit] limit for how many finished
-  transactions can hang around in state
-  @param {function} opts.getNetwork return network number
-  @class
-*/
+ * TransactionStatuses reimported from the shared transaction constants file
+ * @typedef {import('../../../../shared/constants/transaction').TransactionStatuses} TransactionStatuses
+ */
+
+/**
+ * TransactionStateManager is responsible for the state of a transaction and
+ * storing the transaction it also has some convenience methods for finding
```

Comment:
Nit: 
```suggestion
 * storing the transaction. It also has some convenience methods for finding
```

_Sort of_ a pre-existing problem, but the old version at least had a newline that hinted these were separate sentences